### PR TITLE
Add pacemaker-cts test scenario

### PR DESCRIPTION
**Pull request by Evalle**
Pacemaker-CTS thoroughly exercises a pacemaker cluster by running a randomized series of predefined tests on the cluster. We need to add this scenario to make sure that our cloud doesn't have problems with clusterization.

### CODE REVIEW

File: jenkins/ci.suse.de/cloud-mkphyscloud-qa-test-pacemaker-cts.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+          sed -i -e "s,node2,$node2," /tmp/pacemaker_cts_runner
+          sed -i -e "s,node3,$node3," /tmp/pacemaker_cts_runner
+          
+          # run pacemaker-cts on the crowbar node
+          bash /tmp/pacemaker_cts_runner
+          ' 
+          
+          result=$?
+          if [[ $result -gt 0 ]]; then
+            exit $result
```
Comment (aspiers) [inline jenkins/ci.suse.de/cloud-mkphyscloud-qa-test-pacemaker-cts.yaml]: Is it just me or is this missing at least one line?

Comment (Evalle) [inline jenkins/ci.suse.de/cloud-mkphyscloud-qa-test-pacemaker-cts.yaml]: Thanks, scenario has been tested [here](https://ci.suse.de/view/Cloud/view/QA/job/cloud-mkphyscloud-qa-test-pacemaker-cts/97/console) and [here](https://ci.suse.de/view/Cloud/view/QA/job/cloud-mkphyscloud-qa-test-pacemaker-cts/99/console) so it should be fine(not broken), I will fix long lines and the end of the scenario.

**Comment by Evalle** [comment]
Fixed version was tested [here](https://ci.suse.de/view/Cloud/view/QA/job/cloud-mkphyscloud-qa-test-pacemaker-cts/101/console), scenario works fine.

**Comment by Evalle** [comment]
ping @aspiers I fixed my code according to your comments, Fixed version was tested here

**Pull request merged**